test/classes/initializers/miscBlocks.chpl

```diff
@@ -0,0 +1,7 @@
+
+class Foo {
+  proc init() {
+    coforall i in 1..here.maxTaskPar do writeln("Hello from task ", i);
+    coforall loc in Locales do on loc do writeln("Hello from ", here);
+  }
+}
```

Comment:
It looks like your .good file is empty, but these writelns should give output, right?